ci/unit/test_main_node.sh

```diff
@@ -5,6 +5,24 @@
 
 set -ex
 
+remove_daos_path(){
+    file_name=$1
+    while IFS="" read -r LINE || [ -n "$LINE" ]
+    do
+        printf "%s\n" "${LINE//$DAOS_BASE/}" >> "${file_name}.tmp"
+    done < "$file_name"
+
+}
+
+memcheck_convert_xml(){
+    for i in results-*-memcheck.xml; do
+        remove_daos_path "$i"
+    done
+    for i in *.tmp; do
+        mv -- "$i" "${i%%.tmp}"
```

Comment:
You could do this with sed -i couldn't you rather than iterating over the file line by line.

Reply:
I was having problems escaping the / within the DAOS_BASE variable, but now I think I know how to solve this. I'm going to use something like `sed s:$DAOS_BASE::g memcheck-results-1418.xml > memcheck-results-1418.xml.tmp` 
I'll update this in my next patch